homeassistant/components/nad/media_player.py

```diff
@@ -200,12 +200,16 @@ def update(self):
         self._volume = self.calc_volume(self._nad_receiver.main_volume("?"))
         self._source = self._source_dict.get(self._nad_receiver.main_source("?"))
 
-    def calc_volume(self, decibel):
+    def calc_volume(self, decibel: int) -> float:
         """
         Calculate the volume given the decibel.
 
         Return the volume (0..1).
         """
+        # Some receivers cannot report the volume, e.g. C 356BEE,
+        # instead they only support stepping the volume up or down
+        if not decibel:
+            return 1
```

Comment:
instead of returning 1, return `None`. This will prevent the volume level from being displayed while still allowing the volume up and volume down services to work.

Reply:
Ah, thanks! That makes sense :)